test/components/signoff/SimpleReview/PerRecordDiffView_test.jsx

```diff
@@ -135,3 +136,61 @@ describe("findChangeTypes", () => {
     ]);
   });
 });
+
+describe("formatDiffHeader", () => {
+  it("returns expected header based on provided records and displayFields", () => {
+
+    let getTextContent = (props) => {
+      return render(
+        formatDiffHeader(props)
+      ).container.textContent;
+    };
+    
+    expect(getTextContent({
+      target: { id: "foo"}
+    })).toBe("id: foo");
+    
+    expect(getTextContent({
+      source: { id: "foo"}
+    })).toBe("id: foo");
+
+    expect(getTextContent({
+      source: { 
+        id: "foo",
+        prop1: "val1",
+        prop2: "val2",
+      },
+      target: { 
+        id: "foo",
+        prop1: "val3",
+        prop2: "val4",
+      }
+    })).toBe("id: foo");
+
+    expect(getTextContent({
+      source: { 
+        id: "foo",
+        prop1: "val1",
+        prop2: "val2",
+        prop3: "prevVal",
+      },
+      target: { 
+        id: "foo",
+        prop1: "val3",
+        prop2: "val4",
+        // prop3 intentionally undefined
+      },
+      displayFields: [ "prop1", "prop2", "prop3"]
+    })).toBe("prop1: val3prop2: val4prop3: <unknown>id: foo");
```

Comment:
Wouldn't we want `prop3: prevVal` if undefined in target? (genuine question, not sure)
```js
const merged = Object.keys(target).reduce((acc, k) => {
       if (target[k]) { acc[k] = target[k] }
       return acc;
   }, source
);
...
renderDisplayField(merged, f);
```

Reply:
I thought about that and I don't think so, we can easily coalesce to the property rather than the object to do that (or do a little spread magic). This way we're keeping our behavior consistent with the list view by always rendering the current property values, even when undefined. 

We can always switch this later if the users prefer something else.